krux_marathon_api/marathonapi.py

```diff
@@ -71,8 +52,12 @@ def connect(self, address, port):
             s.close()
```

Comment:
Also, use following format for loggers. `self.logger.error("something's wrong with %s:%d.", address, port)` This will allow python interpreter to skip the string substitution if this log is not needed.